system/handlers/admin/datamanager/formbuilder_question.cfc

```diff
@@ -188,6 +199,48 @@ component extends="preside.system.base.AdminHandler" {
 		}
 	}
 
+	private string function preRenderRecord( event, rc, prc, args={} ) {
+		var recordId = prc.recordId ?: ( rc.recordId ?: "" );
+		var isInUse  = formBuilderQuestionService.questionIsInUse( questionId=recordId );
+
+		if ( isTrue( isInUse ) ) {
+			return renderView( view="/admin/formbuilder/_questionInUseWarning", args=args );
+		}
+
+		return "";
+	}
+
+	private void function preDeleteRecordAction( event, rc, prc, args={} ) {
+		var records = args.records ?: QueryNew('');
+
+		for ( var record in records ) {
+			if ( formBuilderQuestionService.questionIsInUse( questionId=record.id ) ) {
+				messageBox.error( translateResource( uri="preside-objects.formbuilder_question:multiAction.question.in.use.warning" ) );
+				setNextEvent( url=event.buildAdminLink( objectName="formbuilder_question" ) );
+			}
+		}
+	}
+
+	private void function extraRecordActionsForGridListing( event, rc, prc, args={} ) {
+		var objectName = args.objectName ?: "";
+		var record     = args.record     ?: {};
+		var recordId   = record.id       ?: "";
+		var isInUse    = formBuilderQuestionService.questionIsInUse( questionId=recordId );
+
+		if ( isTrue( isInUse ) ) {
+			args.actions = args.actions ?: [];
+
+			for ( var action in args.actions ) {
+				if ( find( "deleteRecordAction", action.link ?: "" ) ) {
+					action.link  = "##";
```

Comment:
Might be better to use `link = "javascript:void(0)"` here.

As it stands, the link is still active and will alter the URL. Not a big difference, but ensures the link does absolutely nothing.

(Tip: you can also just say `"javascript:"` and it will have the same effect).

Reply:
Change made when merging PR